src/debian/90-sysctl-hardening.conf

```diff
@@ -21,7 +21,7 @@ kernel.dmesg_restrict = 1
 kernel.kptr_restrict = 2
 
 # Only allow CAP_SYS_ADMIN processes to access "perf" subsystem
```

Comment:
Comment may not be accurate anymore.
Check if 3 wasn't a more secure option.

Reply:
following this document : https://www.kernel.org/doc/Documentation/sysctl/kernel.txt
the highest secure option is ">=2: Disallow kernel profiling by users without CAP_SYS_ADMIN"
2 is specified by bp-028